torch/csrc/jit/function.h

```diff
@@ -39,6 +41,15 @@ struct TORCH_API Function {
     return graph_;
   }
 
+  std::shared_ptr<Graph> optimized_graph() const {
```

Comment:
Nothing is using this yet, so it would be confusing to commit it without hooking it up to the executors.